src/pages/plugins/index.js

```diff
@@ -98,19 +104,35 @@ function PluginComponent({ plugin }) {
   return (
     <div className="row padding-vert--md">
       <div className="col col--10 col--offset-1">
-        <div key={plugin.key} className="card">
-          <div className={classnames('card__header', styles.pluginCardHeader)}>
-            <h2><a className={styles.primaryUrl} href={plugin.primaryUrl}>{plugin.name}</a></h2>
-            <span className="badge badge--primary">{plugin.version}</span>
+        {plugin.deprecatedMessage ? (
+          <div key={plugin.key} className={`card ${classnames('card__deprecated', styles.pluginDeprecatedCard)}`}>
+            <div className={classnames('card__header', styles.pluginCardHeader)}>
+              <h3 className={classnames('deprecated-banner', styles.deprecated)}>Deprecated</h3>
+              <h2><a className={styles.primaryUrl} href={plugin.primaryUrl}>{plugin.name}</a></h2>
+            </div>
+            <div className={classnames('card__flag', styles.pluginCardFlag)}>
```

Comment:
Why do all of these defined a classname that you don't use?  The classnames module should only be used to combine our styles (with CSS Modules) with classes from the Infima framework.

Reply:
This div with class "card_flag" don't need anymore will delete in next pull request